_docs/overview/design-getting-started.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Getting Started
+layout: docs
+category: Overview
+usage: >
+  Read this article to learn how to use the Helix design language, which will help you design faster and provide a more consistent user experience.
```

Comment:
The getting started article doesn't really instruct people on how to use the Helix design language, it really just introduces them to the resources and channels they can take advantage of quickly.

Suggestion: `This article introduces the fundamental concepts and resources a designer can expect to find on the Helix website, as well as the best channels for contacting and collaborating with the Helix team.`

@robb-romans -- any thoughts?

Reply:
@araiford yep, that makes sense

Reply:
updated.